nixos/modules/services/misc/mesos-slave.nix

```diff
@@ -43,6 +51,12 @@ in {
         type = types.bool;
       };
 
+      withDocker = mkOption {
```

Comment:
I think you can `mkDefault cfg.services.docker.enable`.

On Sun, Apr 12, 2015, 12:00 AM Ragnar Dahlén <notifications@github.com>
wrote:

> In nixos/modules/services/misc/mesos-slave.nix
> <https://github.com/NixOS/nixpkgs/pull/7302#discussion_r28198819>:
>
> > @@ -43,6 +51,12 @@ in {
> >          type = types.bool;
> >        };
> >
> > +      withDocker = mkOption {
>
> @wmertens <https://github.com/wmertens> That assumes you want to run the
> docker containerizer as soon as you also have the docker service enabled,
> which is not necessarily true (not in my case anyway). Is there a more
> elegant nix way to do this so that:
>
>    - default is to enable docker containerizer if docker service is
>    enabled
>    - that behaviour is overridable, ie. you can force disabling docker
>    containerizer
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/7302/files#r28198819>.
>